src/item/components/ItemVideoDescription.tsx

```diff
@@ -159,7 +160,7 @@ const ItemVideoDescription: FunctionComponent<ItemVideoDescriptionProps> = ({
 				overflowY: 'auto',
 			}}
 		>
-			<h4 className="c-h4">Beschrijving</h4>
+			<Heading type="h4">Beschrijving</Heading>
```

Comment:
probeer in de toekomst features apart te houden per PR indien mogelijk

Reply:
Will do.